docs/user/tutorials/01-00-x-forwarded-for-header.md

```diff
@@ -0,0 +1,104 @@
+# Forward Client IP in X-Forwarded-For header
+
+Many applications need to know the client IP address of an originating request to behave properly. Usual use-cases include workloads that require the 
+client IP address to restrict their access. The ability to provide client attributes to services has long been a staple of reverse proxies. 
+To forward client attributes to destination workloads, proxies use the X-Forwarded-For (XFF) header. For more information on XFF, see 
+the [IETF’s RFC documentation](https://datatracker.ietf.org/doc/html/rfc7239) and [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#x-forwarded-for).
+
+> **NOTE:** The X-Forwarded-For header is only supported on AWS clusters.
+
+## Prerequisites
+
+* An AWS cluster
+* The Istio module enabled or [Kyma Istio Operator](../../../README.md#install-kyma-istio-operator-and-istio-from-the-latest-release) installed
+* An [Istio Gateway](https://kyma-project.io/#/api-gateway/user/tutorials/01-20-set-up-tls-gateway)
+
+## Steps
+
+### Configure the number of trusted proxies in the Istio custom resource
+
+Applications rely on reverse proxies to forward the client IP address in a request using the XFF header. However, due to 
+the variety of network topologies, you must specify the configuration property **numTrustedProxies**, so that the client address can be extracted correctly. This property indicates the number of trusted proxies deployed
+in front of the Istio Gateway proxy.
+
+Add **numTrustedProxies** to the Istio custom resource (CR):
+
+<!-- tabs:start -->
+#### **kubectl**
+Run the following command:
+
+  ```bash
+  kubectl patch istios/default -n kyma-system --type merge -p '{"spec":{"config":{"numTrustedProxies": 1}}}'
+  ```
+
+#### **Kyma Dashboard**
+1. Navigate to `Cluster Details`.
+2. Open the Istio module's configuration.
+3. Click **Edit**.
+4. Add the number of trusted proxies and click **Update**.
+   ![Add the numTrustedProxies](./assets/01-00-num-trusted-proxies-ui.svg)
+<!-- tabs:end -->
+
+
+### Create a workload for verification
+
+1. [Create a HttpBin workload](https://kyma-project.io/#/api-gateway/user/tutorials/01-00-create-workload).
+2. Expose the HttpBin workload using a VirtualService.
```

Comment:
Last suggestion, could we split this step into two? Like this:
2. Export the following values as environment variables.
 ```bash
    export DOMAIN_TO_EXPOSE_WORKLOADS={GATEWAY_DOMAIN}
    export GATEWAY={GATEWAY_NAMESPACE}/GATEWAY_NAME}
 ```
3. Expose the HttpBin workload using a VirtualService.
 ```bash
    cat <<EOF | kubectl apply -f -
    ...
 ```